src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Index.txt

```diff
@@ -0,0 +1,41 @@
+>> Index([10, 20, 30, 40, 50], 3)
+{Value:30}
+
+>> Index([1, 2, 3, 4, 5], 6)
+#Error
+
+>> Index([1, 2, 3, 4, 5], -1)
+#Error
+
+>> Index([1, 2, 3, 4, 5], Blank())
+Blank()
+
+>> Index(Blank(), 2)
+Blank()
+
+>> Index(Sort([1,2,7,3,4,2],1/Value), 1)
+{Value:7}
+
+>> Index([1, 2, 3, 4, 5], 1/0)
+#Error
+
+>> Index(Table({a:"1", b:"101"}, {a:"2"}, {b:"3"}), 1)
+{a:"1",b:"101"}
+
+>> Index(Filter([1,2,3],Value=4),2)
+#Error
```

Comment:
`#Error(Kind=<the error kind returned by this function>)` - we can now validate the error kinds, and it is good to have it in the tests as the "specification"